.travis.yml

```diff
@@ -11,11 +11,12 @@ before_script:
 
 script:
   - export SBT_OPS=-Xmx4G
-  - if [[ -n "$CODACY_PROJECT_TOKEN" && $TRAVIS_SCALA_VERSION == 2.12* ]]; then sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport coverageAggregate codacyCoverage; else sbt ++$TRAVIS_SCALA_VERSION test; fi
+  - if [[ -n "$CODACY_PROJECT_TOKEN" && $TRAVIS_SCALA_VERSION == 2.12* ]]; then sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport coverageAggregate codacyCoverage; elif [[ $TRAVIS_SCALA_VERSION == 3.0* ]]; then sbt ++$TRAVIS_SCALA_VERSION coreJVM/test coreJS/test circeJVM/test circeJS/test testkitJVM/test testkitJS/test; else sbt ++$TRAVIS_SCALA_VERSION test; fi
```

Comment:
This is ugly, but I don't see any other solution right now. set apparently doesn't properly handle if aggregations have different `crossScalaVersions` 😞: https://github.com/sbt/sbt/issues/3698

Once play-json is published against Scala 3 (PR exists), then one could think of dropping support for spray-json, the project seems pretty dead and users can still use diffson 4.0.3. Then all modules would be aligned again.

Reply:
I plan to switch to GH actions anyway, this will change.